load.py

```diff
@@ -34,15 +34,26 @@
     if os.path.exists(x) and os.path.exists(y):
         full_dataset.append((x, y))
 
+print 'full dataset size : %i' % len(full_dataset)
+
 training_size = int(len(full_dataset) * 0.9)
 val_size = len(full_dataset) - training_size
 train_paths = full_dataset[:training_size]
 val_paths = full_dataset[training_size:]
 print 'num of triaining samples %d' % len(train_paths)
 
+# make sure we don't go off index
```

Comment:
here you want to assert instead of checking

assert val_size < VAL_SAMPLES

I don't see any reason why this should be violated